index.html

```diff
@@ -29,25 +32,192 @@ <h1>
+                  <li>css</li>
+                  <li>javascript</li>
+              </ul>
+          </div>
+          <div class="button">
+              <p>
+                  See Project
+              </p>
+          </div>
+        </div>
+        <div class="grid-item grid-item3">
+            <div class="work-img3">
+
+            </div>
+            <h2>Tonic</h2>
+            <div class="work-content work-content3">
+                <p>CANCOPY</p>
+                <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <div class="work-description">
+              <p class="p-1">
+                  A daily selection of privately personalized reads; no accounts or sign-ups required.
+              </p>
+          </div>
+          <div class="categories">
+              <ul>
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascript</li>
+              </ul>
+          </div>
+          <div class="button">
+              <p>
+                  See Project
+              </p>
+          </div>
+        </div>
+        <div class="grid-item grid-item4">
+            <div class="work-img4">
+
+            </div>
+            <h2>Tonic</h2>
+            <div class="work-content work-content4">
+                <p>CANCOPY</p>
+                <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <div class="work-description">
+              <p class="p-1">
+                  A daily selection of privately personalized reads; no accounts or sign-ups required.
+              </p>
+          </div>
+          <div class="categories">
+              <ul>
+                  <li>html</li>
+                  <li>css</li>
+                  <li>javascript</li>
+              </ul>
+          </div>
+          <div class="button">
+              <p>
+                  See Project
+              </p>
+          </div>
+        </div>
+    </section>
+    <section class="about-myself">
+      <div class="about-header">
+        <h2>About <br> Myself</h2>
+      </div>
+      <div class="about-content">
+        <p class="about-content-p">
+          Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <p class="connect">LET'S CONNECT</p>
+        <a href="twitter.com"><img src="img/twitter.png" alt=""></a>
+        <a href="linkedin.com"><img src="img/Linkedin_icon.png" alt=""></a>
+        <a href="medium.com"><img src="img/medium.png" alt=""></a>
+        <a href="github.com"><img src="img/github.png" alt=""></a>
+        <a href="angellist.com"><img src="img/angellist.png" alt=""></a>
+        <button type="button" class="my-button">Get My Resume</button>

```

Comment:
- [ ] kind use an `ul tag` here and for each icon, use a `li`

Reply:
Implemented 👍